src/main/java/com/dkohut/chatservice/common/interfaces/IMessageDAOService.java

```diff
@@ -0,0 +1,40 @@
+package com.dkohut.chatservice.common.interfaces;
+
+import java.util.List;
+
+import com.dkohut.chatservice.common.entity.Message;
+
+/**
+ * 
+ * @author Dmytro Kohut
+ *
+ */
+public interface IMessageDAOService {
+
+	/**
+	 * Select messages of one chat in given range, sorted by created_at column.
+	 * 
+	 * @param Integer firstRow
+	 * @param Integer numberOfRows
+	 * @param Integer chatRoomId
+	 * @return List<Messages> 
+	 */
+	List<Message> select(Integer firstRow, Integer numberOfRows, Integer chatRoomId);
+	
+	/**
+	 * Create new message.
+	 * 
+	 * @param Message message
+	 * @return Integer id
+	 */
+	Integer create(Message message);
+	
+	/**
+	 * Delete existing message.
+	 * 
+	 * @param Integer id
+	 * @return String
+	 */
+	String delete(Integer id);
```

Comment:
Not clear why we return `String`

Reply:
It was a bad idea. The return type was changed.